CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ ___Note:__ Yet to be released changes appear here._
 
 * `BpmnRenderer` only renders actual flow elements (e.g. `bpmn:IntermediateCatchEvent` but not `bpmn:MessageEventDefinition`)
 
+## 14.2.0 (2023-10-31)
```

Comment:
Why does it say `14.2.0` here?

Reply:
Because release-please doesn't like manual releases. v14.2.0 was released manually without GitHub Release which messed up the tool.